packages/test/src/helpers.ts

```diff
@@ -98,6 +98,21 @@ export function cleanStackTrace(ostack: string): string {
   return normalizedStack ? `${firstLine}\n${normalizedStack}` : firstLine;
 }
 
+/**
+ * Compare stack traces using $CLASS keyword to match any inconsistent identifiers
+ *
+ * As of Node 24.6.0 type names are now present on source mapped stack traces which leads
+ * to different stack traces depending on Node version.
+ * See [f33e0fcc83954f728fcfd2ef6ae59435bc4af059](https://github.com/nodejs/node/commit/f33e0fcc83954f728fcfd2ef6ae59435bc4af059)
+ */
+export function compareStackTraceIdentifiers<T>(t: ExecutionContext<T>, actual: string, expected: string): void {
```

Comment:
Not sure about this function's name. Why `...Identifiers`?

Also, can probably avoid the `T` generic parameter, this function never depends on that type. Try accepting `ExcutionContext` instead, or `ExcutionContext<unknown>` if the parameter is required (it is not AFAIR).

Reply:
I am not especially happy with the name. I wasn't sure if `compareStackTrace` was too generic.

Reply:
`compareStackTrace` seems fine to me. And that's an internal, test-only function anyway, so we can reconsider in the future if we ever feel it needs to be more specific.